src/_includes/about.njk

```diff
@@ -0,0 +1,7 @@
+<section class="lg:mt-24 md:mt-2 lg:-pr-32 pl-24 text-light relative inline-block md:mr-24 lg:mr-40 pointer-events-none">
+  
+  <div class="text-seafoam text-3xl pb-6">About Us</div>
+  <div class="relative w-full leading-10 text-justify text-seafoam text-xs xl:text-s
+  pointer-events-auto md:tracking-wide"> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{{ event_info.about_info }}</div>
```

Comment:
`&nbsp;` wasn't intended to be used for indentation. Can we find a css-only solution for this? We might need to add another Tailwind plugin, like [tailwindcss-typography](https://www.npmjs.com/package/tailwindcss-typography).

Reply:
Resolved with new css class "indent" registered under Tailwind Utilities